src/com/rockbite/bootcamp/util/command/CommandManager.java

```diff
@@ -0,0 +1,102 @@
+package com.rockbite.bootcamp.util.command;
+
+import com.rockbite.bootcamp.Player;
+import com.rockbite.bootcamp.Product;
+import com.rockbite.bootcamp.util.pool.Pool;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * this class Manages commands
+ */
+public class CommandManager {
+    //history of commands
+    List<Command> history = new ArrayList<>();
+    //cursor shows when we are in the history
+    private int cursor = 0;
```

Comment:
cursor for what? In this simple class this is ok, but as you expand this it may start to be unclear as to what this cursor is used for.

historyCursor would be a good name for this var.